src/main/java/unicash/model/UniCash.java

```diff
@@ -95,6 +97,40 @@ public void removeTransaction(Transaction key) {
         transactions.remove(key);
     }
 
+    private List<Transaction> getAllExpenses() {
+        return getTransactionList()
+                .stream()
+                .filter(t -> t.getType().toString().equals("expense"))
```

Comment:
This filter clause should be done as `t.getType().getType().equals(TransactionType.EXPENSE)`, this way, in case the enum value is ever changed, this won't cause weird regressions!

Reply:
Thanks for the suggestion!